app/utils/db.server.ts

```diff
@@ -34,3 +36,9 @@ export const prisma = remember('prisma', () => {
 	client.$connect()
 	return client
 })
+
+const recordMetrics = (e: Prisma.QueryEvent) => {
+	prometheus.sqlQueryCounter.inc({ query: e.query })
+	prometheus.sqlQueryDuration.observe({ query: e.query }, e.duration)
+	prometheus.sqlQueryDurationSummary.observe({ query: e.query }, e.duration)
+}
```

Comment:
Since this is only called in one place I think I'd rather inline it above instead of have this function.